docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -0,0 +1,143 @@
+=============================            ================================================
+
+
+Settings used by scheduler
+--------------------------
+
+The following etcd keys are used by the scheduler module of the controller component.
+
+====================================      ===============================================
+setting                                   description
+====================================      ===============================================
+/deis/controller/schedulerModule          scheduler backend, either "fleet" or "swarm"
+                                          (default: "fleet")
+====================================      ===============================================
+
+
+Fleet Scheduler
+---------------
+
+`fleet`_ is a scheduling backend included with CoreOS:
+
+    fleet ties together systemd and etcd into a distributed init system. Think of
+    it as an extension of systemd that operates at the cluster level instead of the
+    machine level. This project is very low level and is designed as a foundation
+    for higher order orchestration.
+
+``fleetd`` is already running on the machines provisioned for Deis: no additional
+configuration is needed. Commands such as ``deis ps:restart web.1`` or
+``deis scale cmd=10`` will use `fleet`_ by default to manage app containers.
+
+To use the Fleet Scheduler backend explicitly, set the controller's
+``schedulerModule`` to "fleet":
+
+.. code-block:: console
+
+    $ deisctl config controller set schedulerModule=fleet
+
+
+Swarm Scheduler
+---------------
+
+.. important::
+
+    The Swarm Scheduler is a technology preview and is not recommended for
+    production use.
+
+`swarm`_ is a scheduling backend for Docker:
+
+    Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
+    into a single, virtual host.
+
+..
+
+    Swarm serves the standard Docker API, so any tool which already communicates
+    with a Docker daemon can use Swarm to transparently scale to multiple hosts...
+
+Deis includes an enhanced version of swarm v0.2.0 with node failover and optimized
+locking on container creation. The Swarm Scheduler uses a `soft affinity`_ filter
+to spread app containers out among available machines.
+
+Swarm requires the Docker Remote API to be available at TCP port 2375. If you are
+upgrading an earlier installation of Deis, please refer to the CoreOS documentation
+to `enable the remote API`_.
+
+.. note::
+
+    **Known Issues**
+
+    - It is not yet possible to change the default affinity filter.
+    - If swarm can't create all the containers requested, Deis returns an error
+      and leaves some containers in the "created" state. Until this behavior is fixed,
+      be mindful of resource limitations on your cluster.
+
+To test the Swarm Scheduler backend, first install and start the swarm components:
+
+.. code-block:: console
+
+    $ deisctl install swarm && deisctl start swarm
```

Comment:
> this didn't work

Could you be more specific? `deisctl install swarm` is what we've been using--did you rebuild `deisctl` from this PR branch?